### PR TITLE
FAQ issue resolved

### PR DESCRIPTION
**Pull request by ajay-sai**
This reads the questions and answers from a table which can be created using this: 
CREATE TABLE faq (id int(11) PRIMARY KEY AUTO_INCREMENT, question text, answer text);

Let me know if you need any other changes in the FAQ page

### CODE REVIEW

File: application/controllers/faq.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * @copyright  Ushahidi - http://www.ushahidi.com
+ * @license	   http://www.gnu.org/copyleft/lesser.html GNU Lesser General Public License (LGPL)
+ */
+
+class Faq_Controller extends Main_Controller {
+	function __construct()
+	{
+		parent::__construct();
+
+//		$this->template->this_page = 'faq';
```
Comment (Neetuj) [inline application/controllers/faq.php]: please remove the comment/dead code

File: application/controllers/faq.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+			->find_all((int) Kohana::config('settings.items_per_page_admin'), $pagination->sql_offset);
+			
+		$this->template->content->messages = $messages;
+		$this->template->content->pagination = $pagination;
+
+		// Total Questions
+		$this->template->content->total_items = $pagination->total_items;
+		
+		// Javascript Header
+//		$this->themes->js = new View('members/private_js');
```
Comment (Neetuj) [inline application/controllers/faq.php]: please remove the not-useful comments/dead code

File: application/helpers/members.php [hunk 3]
```diff
@@ -97,7 +97,7 @@ public static function alerts_subtabs($this_sub_page = FALSE)
         // Action::nav_members_alerts - Add items to the members alerts navigation tabs
         Event::run('ushahidi_action.nav_members_alerts', $this_sub_page);
     }
-	
+
```
Comment (Neetuj) [inline application/helpers/members.php]: unnecessary

File: application/helpers/members.php [hunk 4]
```diff
@@ -128,4 +128,4 @@ public function gravatar($email, $s = 80, $d = 'mm', $r = 'g', $img = FALSE, $at
 		}
 		return $url;
 	}
-}
\ No newline at end of file
+}
```
Comment (Neetuj) [inline application/helpers/members.php]: I am assuming there was a syntax error(missing bracket) which was added to fix the issue

File: application/models/faq.php [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ * @author     Ushahidi Team <team@ushahidi.com> 
+ * @package    Ushahidi - http://source.ushahididev.com
+ * @subpackage Models
+ * @copyright  Ushahidi - http://www.ushahidi.com
+ * @license    http://www.gnu.org/copyleft/lesser.html GNU Lesser General Public License (LGPL) 
+ */
+
+class Faq_Model extends ORM
+{
+//	protected $belongs_to = array('user_id');
```
Comment (Neetuj) [inline application/models/faq.php]: please remove the not-useful comments/dead code

File: sql/ushahidi.sql [hunk 6]
```diff
@@ -1145,6 +1145,21 @@ CREATE TABLE IF NOT EXISTS `private_message` (
 -- --------------------------------------------------------
 
 /**
+ * Table structure for table `faq`
+ *
+ */
+
+CREATE TABLE IF NOT EXISTS `faq` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
```
Comment (Neetuj) [inline sql/ushahidi.sql]: where does this number 11 come from ?  was is stated in the requirements/specs that id will be this long ?

**Comment by rosariorobinson** [comment]
Where are we with this PR? Can I merge?

**Comment by jproft** [comment]
The FAQ page requirement was dropped when we showed the PCVs how to create an FAQ page using the built-in functionality Ushahidi has for adding pages, so this PR is not needed. I'll close this.

**Pull request closed without merging**